### ISSUE TITLE
clean up code

### ISSUE THREAD

**Issue by jdittrich** [jdittrich/ep_autocomp#29]
The code  in `https://github.com/jdittrich/ep_autocomp/blob/master/static/js/autocomp.js` is messy. 

It is commented, but the order of the functions does not make sense and the comments should be reviewed. 

* [ ] moveUp and moveDown functions should be adjacent 
* [ ] createAutocompHTML does return undefined, not ?

**Comment by lpagliari** [comment]
> createAutocompHTML does return undefined, not ?

Yes. :-) It does not return anything, it is used to display new suggestions to the user.
